src/panels/PanelBase.js

```diff
@@ -97,41 +97,45 @@ export default {
   },
   methods: {
     toggle() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
-        this.isCached = true;
-      }
-    },
-    expand() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
+      if (!this.isCached && !this.localExpanded) {
```

Comment:
Content is not loaded and not expanded.
We load the content first to get the max height, then play the animation